zds/tutorialv2/tests/tests_views/tests_removeauthor.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import patch
+
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.test import TestCase
+from django.urls import reverse
+
+from zds.gallery.models import UserGallery
+from zds.member.tests.factories import ProfileFactory
+from zds.tutorialv2.models.database import PublishableContent
+from zds.tutorialv2.tests import override_for_contents, TutorialTestMixin
+from zds.tutorialv2.tests.factories import PublishableContentFactory
+
+
+@override_for_contents()
+class RemoveAuthor(TutorialTestMixin, TestCase):
+    def setUp(self):
+        self.bot_group = Group(name=settings.ZDS_APP["member"]["bot_group"])
+        self.bot_group.save()
+
+        self.overridden_zds_app["member"]["bot_account"] = ProfileFactory().user.username
+
+        self.author = ProfileFactory().user
+        self.other_autor = ProfileFactory().user
```

Comment:
5 occurrences à renommer
```suggestion
        self.other_author = ProfileFactory().user
```

Reply:
C'est corrigé.